package.json

```diff
@@ -5,20 +5,22 @@
   "author": "David Peter <mail@david-peter.de>",
   "license": "MIT",
   "type": "module",
+  "engines": {
+    "node": ">=10"
+  },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/sharkdp/insect.git"
   },
   "homepage": "https://github.com/sharkdp/insect",
   "bin": {
-    "insect": "./index.js"
+    "insect": "./index.min.cjs"
```

Comment:
What do you think about finding better filenames for `index.js` and `index.min.cjs`? Maybe `index.dev.js` (feels a little wrong to me since it's compiled to the production version in the end) and `index.cjs` respectively?

Reply:
Okay for me.